v2/apiserver/internal/api/projects.go

```diff
@@ -95,6 +95,15 @@ func (p ProjectList) Contains(project Project) bool {
 	return false
 }
 
+// ProjectUpdateOptions represents useful, optional settings for updating a
+// Project. It currently has no fields, but exists to preserve the possibility
```

Comment:
Note to remove "It currently has no fields..."

Reply:
Fixed in f02fbd3faa48ff69152a01a464105a9745887d22